request.cc

```diff
@@ -0,0 +1,120 @@
+#include "request.h"
+
+#include <boost/tokenizer.hpp>
+#include <boost/algorithm/string.hpp>
+
+using Headers = std::vector<std::pair<std::string, std::string>>;
```

Comment:
Thanks for the catch! Not needed, but I do need to replace `Headers` with `Request::Headers` now